src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -275,6 +279,14 @@ const LostAndFound = () => {
                     <div className={styles.dataCell}>{report.description}</div>
                   </Grid>
                 </Grid>
+                {/* Show notification for "found" status */}
+                {report.status.toLowerCase() === 'found' && (
```

Comment:
I dont think making the found green on the main page really achieves the consistency we want, especially because the past items, look different, and if you have some active items, those would look different being in the same active card and all that on the main page.
Also I think having a different card for just Found items, is kinda unnecessary, since they could all just live under the Active Reports card. But again if you feel strongly about it, i dont mind changing it, but from my view and i think i can say Eriks, it seemed fine.

Reply:
I think the key point is that we want it to be obvious, the whole reason behind this feature is to remind people that their item is found, so having it be as a separate section, with a notification badge (the number in circle) makes it seem like a separate notification, and matches the design on the home page widget.  The purpose of the green is to be a specific notification, so past and active aren't really an update to the report, so the whole point is to be inconsistent with the others, to catch your attention.